orchestrator/security.py

```diff
@@ -36,8 +33,10 @@
 )
 
 
-async def authenticate(request: Request) -> OIDCUserModel | None:
-    return await request.app.auth_manager.authentication.authenticate(request)
+async def authenticate(
+    request: Request, token: Annotated[str | None, Depends(TokenExtractor())] = None
```

Comment:
```suggestion
    request: Request, token: Annotated[str | None, Depends(HttpBearerExtractor.extract)] = None
```

Why not this? am I missing something? So we don't need your PR on the oauth-lib?

because I think we had this `HttpBearerExtractor` in oauth-lib exactly for this. Hope I am not wrong

Reply:
of course, you have to instantiate `HttpBearerExtractor` or you can make it a static method in base class